pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -206,7 +208,11 @@ let
       build --action_env=PYENV_ROOT
       build --python_path="${python}/bin/python"
       build --distinct_host_configuration=false
-      build --define PROTOBUF_INCLUDE_PATH="${protobuf}/include"
+      build --define PROTOBUF_INCLUDE_PATH="${pkgs.protobuf}/include"
+    '' + lib.optionalString (stdenv.targetPlatform.isx86_64 && stdenv.targetPlatform.isUnix) ''
+      build --config=avx_posix
```

Comment:
This should be dependent on stdenv.hostPlatform.avxSupport

Reply:
ah didn't know that existed! would that be 
```
    '' + lib.optionalString (stdenv.targetPlatform.avxSupport && stdenv.targetPlatform.isUnix) ''
```
then?

Reply:
done in https://github.com/NixOS/nixpkgs/pull/237648/commits/d2f59991294188e83b9a61c92045fbbea364f37b